demisto_sdk/main.py

```diff
@@ -169,15 +170,32 @@ def unify(**kwargs):
 @click.option(
     '-g', '--use-git', is_flag=True, show_default=True,
     default=False, help='Validate changes using git - this will check your branch changes and will run only on them.')
+@click.option(
+    '-p', '--path', help='Path of file to validate specifically.'
+)
+# @click.option(
+#    '-t', '--file-type', type=click.Choice(
+#        ['integration', 'script', 'playbook', 'incidnet-field', 'indicator-field', 'image', 'release-notes', 'layout',
+#         'description']), help='If you want to validate a specific file, you can specify it\'s type.')
 @pass_config
 def validate(config, **kwargs):
     sys.path.append(config.configuration.env_dir)
 
-    validator = FilesValidator(configuration=config.configuration,
-                               is_backward_check=str2bool(kwargs['backward_comp']),
-                               is_circle=str2bool(kwargs['circle']), prev_ver=kwargs['prev_ver'],
-                               validate_conf_json=kwargs['conf_json'], use_git=kwargs['use_git'])
-    return validator.run()
+    # if not kwargs.get('path') and kwargs.get('file_type'):
+    #    click.echo(click.style('Ignoring file-type argument since no specific file was determined', fg='yellow'))
```

Comment:
?

Reply:
It was there in case we wanted to pre-define the type, but I just added the logic to the regular flow so it is not needed